monkey/infection_monkey/exploit/powershell_utils/credentials.py

```diff
@@ -36,15 +36,17 @@ def get_credentials(
 
 
 # On Windows systems, when username == None and password == None, the current user's credentials
-# will be used to attempt to log into the victim.
+# will be used to attempt to log into the victim only on the first hop, from island
+# to a machine. Propagating after the first hop is not possible at the moment.
+# See this: https://security.stackexchange.com/questions/120422/powershell-and-cached-credentials
```

Comment:
How is this comment relevant to the code below? I feel like this should go to PowerShell exploiter description or documentation hub

Reply:
As there is an issue for this, I thought it is relevant to put there. But I agree it should go into the description or documentation hub.